### ISSUE TITLE
Integration tests do not have HCP

### ISSUE THREAD

**Issue by merrygoat** [royallgroup/TCC#87]
Get a more crystalline structure for one of the unit tests - currently there is no HCP.

A deeply supercooled Wahnstrom would be good too as the current tests have lots of 4 fold order but not much 5 fold (10B, 13A).

Once this is in place - check the work already done on HCP to make sure nothing has been broken.
